### DISCUSSION TITLE
Showing one resource at a time in our FE UI

### DISCUSSION THREAD

**Discussion by Utkarshbhimte** [covidarmy/backend#39] [Ideas]
I have used the https://www.introbot.co/ WA chatbot and I like the idea of showing one resource at a time and asking for the status to move to the next resource, 

It decreases cognitive load to the user and it also helps us to get the status for any resource.

**Comment by rgxai** [comment]
Good idea. We are doing this for our own WA chatbot too (1 resource at a time). We should definitely try this but it is also linked to the prompt flow [here](https://github.com/covidarmy/backend/issues/36). So maybe we can take a final call on both and merge into a single issue.
